src/main/java/com/salesforce/mc/identity/common/schema/CBKDataPathRequest.java

```diff
@@ -0,0 +1,46 @@
+package com.salesforce.mc.identity.common.schema;
+
+import java.util.List;
+
+/**
+ * Schema of the CBK Data path's request object
+ * @author Tej Vepa
+ *
+ */
+public class CBKDataPathRequest {
+
+	private final String orgUUID;
+	private final String basePath;
+	private final List<String> bridgeKeys;
+	private final List<String> sourceTypes;
+	private final String dbMode;
+
+	public CBKDataPathRequest(final String orgUUID, final String basePath, final List<String> bridgeKeys,
+			final List<String> sourceTypes, final String dbMode) {
+		this.orgUUID = orgUUID;
+		this.basePath = basePath;
+		this.bridgeKeys = bridgeKeys;
+		this.sourceTypes = sourceTypes;
+		this.dbMode = dbMode;
```

Comment:
what are the possible value of dbmode?

Reply:
console_prod etc

Reply:
depending on what db we want to connect to.